jenkins/gradle/gradle-check.jenkinsfile

```diff
@@ -111,7 +113,7 @@ pipeline {
                         if (!env.BUILD_CAUSE.contains('Started by user') && !env.BUILD_CAUSE.contains('Started by timer')) {
                             def pr_url = "${pr_to_clone_url}".replace(".git", "/pull/${pr_number}")
                             println("Triggered by GitHub: ${pr_to_clone_url}")
-                            if ("$pr_number" == "Null") {
+                            if ("$post_merge_action" == "true") {
```

Comment:
With this change https://github.com/opensearch-project/OpenSearch/pull/13786/files the `pr_number` is not null anymore. A new field `post_merge_action` with true/false is added to identify the `push` event.